R/s2-lnglat.R

```diff
@@ -80,14 +80,14 @@ as.matrix.s2_lnglat <- function(x, ...) {
 
 #' @importFrom wk as_wkb
 #' @export
-as_wkb.s2_lnglat <- function(x, ..., endian = wk::wk_platform_endian()) {
-  as_wkb(as_s2_geography(x), ..., endian = endian)
+as_wkb.s2_lnglat <- function(x, ...) {
```

Comment:
That is fine, but breaks `sf` on revdep checks, so can only go to CRAN after an sf update.